MobileFilter/sections/css_extended.txt

```diff
@@ -3,6 +3,8 @@
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/59614
 mensxp.com#?#.list-card > li:not(.track_ga):has(> div[class*=" "] > div > div > a[onclick])
+! https://github.com/AdguardTeam/AdguardFilters/issues/60338
+m.vk.com#?#.wall_posts > div.wall_item:has(> div.wi_body > div.ads_mark)
```

Comment:
On my end - element between `.wall_posts` and `div.wall_item`
![image](https://user-images.githubusercontent.com/8361299/89108164-f1528780-d43e-11ea-8b31-569d5cc8c893.png)

Reply:
I checked my rule, and it worked for me. I'll check again.

Reply:
just make `.wall_posts div.wall_item`

Reply:
Ok
![Screenshot 2020-08-01 at 22 34 58](https://user-images.githubusercontent.com/18536328/89109160-5d38ee00-d447-11ea-98ac-ee5d47aa19d3.png)